### PR TITLE
Adding GenerateWP

### PR DESCRIPTION
**Pull request by ramiy**
# Thank you for opening a pull request to tiny-helpers.dev ❤️

Before you submit your PR, please take a look at the [contributing guidelines](https://github.com/stefanjudis/tiny-helpers/blob/master/CONTRIBUTING.md). :)

To make it easier for us to review and merge your PR, please make sure …

- [*] you only add one (!) new helper per pull request.
- [*] you have checked if an open PR already exists.
- [ ] the submitted website is focused on a single, development related issue.
- [*] the `desc` field includes an "actionable sentence" (e.g. "Create something great" or "Transform something into something else").

Thank you! 🙇‍♂️

### CODE REVIEW

**Comment by ramiy** [comment]
@stefanjudis

Actually login is not required. You have 28 free tools. You don't have to login to use them.

**Comment by stefanjudis** [comment]
Aah. I missed that because the UI is not very clear there. I'm happy to take it (but now the branch is gone). :)

**Comment by ramiy** [comment]
@stefanjudis branch restored.

File: helpers/generatewp.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "GenerateWP.com",
+  "desc": "User friendly tools for WordPress developers.",
```
Comment (stefanjudis) [inline helpers/generatewp.json:3]:
```suggestion
  "desc": "Find tools for WordPress developers.",
```

**Comment by stefanjudis** [comment]
Thanks. :)

**Pull request merged**